web_development_101/javascript_basics/project_rock_paper_scissors.md

```diff
@@ -41,12 +41,17 @@ Submit a solution with a pull request to this [file](https://github.com/TheOdinP
 
 - Add your solution below this line!
 - [CoderOO7 solution (with GUI)](https://github.com/coderoo7/rockPaperScissor) - [View in Browser](https://coderoo7.github.io/rockPaperScissor)
+>>>>>>> 5e1cfee47336aeec36ff2c68e43c5b5f1d0a44e9
```

Comment:
It looks like these merge conflict markers were left behind, can you remove them please? any line that starts with`>>>>` will be merge conflict markers.

Reply:
removed markers thanks